### DISCUSSION TITLE
Disable when coding vue project in neovim?

### DISCUSSION THREAD

**Discussion by ogios** [typescript-language-server/typescript-language-server#748] [Q&A]
Is there a way to stop loading it when I'm in a vue project?
for example, when detected vite.config.js/ts or vue.config.js/ts, do not load tsserver
or if there's a volar lsp running, do not load tsserver

is it possible to configure nvim lspconfig like this?

**Comment by rchl** [comment]
I think you should look for help on how to solve it within the neovim community. This server is not gonna add logic to find arbitrary configuration files of other servers.
